Submission PhoneBook project

antonderegt opened the pull request:
Hi Reviewer,

I'd love to hear your thoughts!

Doc415 reviewed (changes requested):
Hey again. 
We have some issues here to fix :

1- Requested phone number format does not fits the standarts. For example:
 I have phone number
+90 312 222 33 11
and you are validating it for 9 numbers after + so it fails. If i were living in USA it will be
+1 312 22 33 11 so the numbers count changes. 
Maybe you can use another algorithm or a library for that.

2-When i am updating a record , it only allows to change one property and returns to main menu, i had to do it three times if i want to change all properties of the record

3- when i choose update before adding any records, application crashes :)
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Phonebook/assets/94632226/b9814301-7c06-45f2-990c-afec317162f1)

Suggestion:
It would be nice to start working on async CRUD operations. (not a requirement)

antonderegt commented:
Woohoo, thanks @Doc415 great review. I especially enjoyed the suggestion about async!

Fixed problem 1 with the package libphonenumber-csharp.
Fixed problem 2 by asking the user if they want to update another property.
Fixed problem 3 by validating if there are contacts in the database. Fixed for updating and removing.

Thanks

Doc415 reviewed (approved):
Looks much better.
Approving your project.
